app/views/developer_skills/new.html.erb

```diff
@@ -1,4 +1,4 @@
 <div class="container p-5 w-50">
-  <h1>Add a skill to your profile</h1>
+  <h1>Add skills to your profile</h1>
```

Comment:
There are quite a lot of skills in the form that repeat, e.g. wordpress is there 3 times